src/animals/animals.resolver.ts

```diff
@@ -0,0 +1,38 @@
+import { Resolver, Query, Mutation, Args } from '@nestjs/graphql';
+import { AnimalsService } from './animals.service';
+import { Animal } from './entities/animal.entity';
+import { CreateAnimalInput } from './dto/create-animal.input';
+import { UpdateAnimalInput } from './dto/update-animal.input';
+
+@Resolver(() => Animal)
+export class AnimalsResolver {
+  constructor(private readonly animalsService: AnimalsService) {}
+  @Mutation(() => Animal)
+  createAnimal(
+    @Args('createAnimalInput') createAnimalInput: CreateAnimalInput,
+  ) {
+    return this.animalsService.create(createAnimalInput);
+  }
+
+  @Query(() => [Animal], { name: 'AllAnimals' })
+  findAll() {
+    return this.animalsService.findAll();
```

Comment:
method findAll does not exist on animal Service, please create it

Reply:
fixed